charts/ssi-credential-issuer/templates/job-issuer-migrations.yaml

```diff
@@ -66,14 +66,59 @@ spec:
           - name: "SEEDING__TESTDATAENVIRONMENTS__0"
             value: "{{ .Values.migrations.seeding.testDataEnvironments }}"
           - name: "SEEDING__DATAPATHS__0"
-            value: "{{ .Values.migrations.seeding.testDataPaths }}"
+            value: "Seeder/Data"
+          {{- if (.Values.migrations.seeding.testData.useDefaultTestData) }}
+          - name: "SEEDING__DATAPATHS__1"
+            value: "Seeder/Data/import/default"
+          - name: "SEEDING__TESTDATAENVIRONMENTS__1"
+            value: "test"
```

Comment:
maybe we shouldn't call the files `test` as that likely the identifier which would be used in the additional configMap, maybe `defaulttest`

Reply:
adjusted